addons/mail/static/src/core/web/activity_list_popover_item.xml

```diff
@@ -29,10 +29,10 @@
                 </button>
             </div>
             <div class="d-flex align-items-center flex-wrap mx-3">
-                <img t-if="props.activity.user_id[0]" class="me-2 rounded o_object_fit_cover" t-att-src="activityAssigneeAvatar" style="max-width: 1.5rem; max-height: 1.5rem;"/>
+                <img t-if="props.activity.persona.userId" class="me-2 rounded o_object_fit_cover" t-att-src="props.activity.persona.avatarUrl" style="max-width: 1.5rem; max-height: 1.5rem;"/>
                 <div class="mt-1">
-                    <t t-if="props.activity.user_id[0]">
-                        <small class="text-truncate" t-esc="props.activity.user_id[1]"/>
+                    <t t-if="props.activity.persona.userId">
```

Comment:
I don't think we need to keep a check on `userId` in this template if we no longer use `userId`.
Maybe a check on `persona`, but I'd argue we should always send it in data with activity.

Reply:
If there is always a persona with the activity, I guess it's not useful to check it at all